ras_party/controllers/account_controller.py

```diff
@@ -291,6 +291,15 @@ def request_password_change(payload, session):
     return {'response': "Ok"}
 
 
+@with_db_session
+def change_respondent_account_status(payload, party_id, session):
+
+    status = payload['status_change']
+
+    respondent = query_respondent_by_party_uuid(party_id, session)
```

Comment:
Currently if party does not exist this raises a 500 error. Do we need to cover this (and return 404)? (And should we have a test for it?)